doc/nrf/ug_multi_image.rst

```diff
@@ -4,9 +4,27 @@ Multi-image builds
 ##################
 
 In many cases, the firmware that is programmed to a device consists of not only one application, but several separate images, where one of the images (the *parent image*) requires one or more other images (the *child images*) to be present.
+The child image then *chain-loads* (or *boots*) the parent image, which in turn might be a child image to another parent image and boot that one.
 The most common use case for builds consisting of multiple images is an application that requires a bootloader to be present.
 
-In the context of |NCS|, multiple images are required in the following scenarios:
+
+When to use multiple images
+***************************
+
+An *image* (also referred to as executable, program, or elf file) consists of pieces of code and data that are identified by image-unique names recorded in a single symbol table.
+The symbol table exists as metadata in a ``.elf`` or ``.exe`` file and is not included when the image is converted to a HEX file for programming.
+Instead, the code and data are placed at addresses by a linker.
+This linking process is what distinguishes images from object files (which do not require linking).
+Therefore, to determine if you have zero, one, or more images, count the number of times the linker runs.
+
+Using multiple images has the following advantages:
+
+* You can run the linker multiple times and partition the final firmware into several regions.
+  This partitioning is often useful for bootloaders.
+* Since there is a symbol table for each image, the same symbol names can exist multiple times in the final firmware.
+  This is useful for bootloader images, which might required their own copy of a library that the application uses, but in a different version or configuration.
+
+In the |NCS|, multiple images are required in the following scenarios:
```

Comment:
Is it more correct to say "In NCS"?

I consider NCS to be a name, not a concept or even an abbreviation (In the Nordic Connect SDK does sound right).

So, to me it sounds like saying "In the Google", or "In the Facebook."

Reply:
|NCS| is a shortcut for nRF Connect SDK, so this will resolve to "In the nRF Connect SDK".
We don't use the abbreviation NCS.

Reply:
aha